bookops_marc/bib.py

```diff
@@ -101,6 +101,27 @@ def cataloging_date(self) -> Optional[date]:
         else:
             return None
 
+    @property
+    def collection(self) -> Optional[str]:
+        """
+        Returns the collection that the record is a part of.
+
+        Options are "RL" (for research materials), "BL" (for branch materials),
+        "mixed", or None. An NYPL record will have the collection "RL" or "BL" if the
+        record contains one 910$a field with the value "RL" or "BL". If an NYPL record
+        contains two 910$a fields and both "BL" and "RL", the record's collection will
+        be "mixed". All other records will be assigned `None` as the collection type.
+        """
+        if not self.library == "nypl":
+            return None
+        subfields = [i.get("a") for i in self.get_fields("910") if i]
```

Comment:
It would be good to strip whitespace in value of the subfield $a. On occasion these fields are added manually and such things may occur.